src/postprocess/band_structure.jl

```diff
@@ -113,7 +113,7 @@ function plot_band_data(band_data; εF=nothing,
             energies = (data.λ[ibranch][spin][iband, :] .- eshift) ./ unit_to_au(unit)
 
             color = (spin == :up) ? :blue : :red
-            Plots.plot!(p, kdistances, energies, color=color, label="", yerror=yerror)
+            Plots.plot!(p, kdistances, energies, color=color, label="", yerror=yerror, marker=4)
```

Comment:
I prefer to use symbols here.

Reply:
how do you do that?

Reply:
4 is just the size, and it puts a nice round so I think that's OK?

Reply:
ah so it's markersize ... yes plots maps these keys around. But for library code the long form is favoured.

Reply:
So I'm confused, you want `marker=:o`?

Reply:
markersize=4 would be what I want ;)

Reply:
Is what is now OK?

Reply:
Actually for the default settings using this flag looks pretty crap. I'd suggest to use `markershape=:+` and only if the density of kpoints is not too large.

Reply:
I don't like the + because the directionality just looks weird. I can decrease markersize if you want? I don't particularly care just edit the line with what you want